### PR TITLE
[DATA-2658] Fix bugs in helm chart and use ingress

### PR DESCRIPTION
**Pull request by JB-SendBird**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Review by EJSohn — APPROVED** [review]
LGTM! Let's bring it to the meeting with infra team

File: chart/values_nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ingress:
+  web:
+    annotations:
+      alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:ap-northeast-1:242864343139:certificate/d6930331-0470-4d85-90c4-7b095914bf1f
```
Comment (JB-SendBird) [inline chart/values_nightly.yaml:7]: Created with Terraform. We need to discuss how to store & manage this kind of value further.

File: chart/values.yaml [hunk 2]
```diff
@@ -660,6 +669,13 @@ config:
     enable_proxy_fix: 'True'
     expose_config: 'True'
     rbac: 'True'
+    authenticate: 'True'
+    auth_backend: 'airflow.contrib.auth.backends.google_auth'
+  google:
+    client_id: ''
+    client_secret: ''
+    oauth_callback_route: '/oauth2callback'  # this doesn't have any effect if rbac is turned on
```
Comment (JB-SendBird) [inline chart/values.yaml:677]: When we use `rbac` feature of airflow, this value doesn't have effect. The callback path (redirect path) is always set `oauth-authorized/<provider>`, that is, `oauth-authorized/google` so we need to register the path to "Authorized redirect URIs" of Google OAuth api.

File: chart/values_nightly.yaml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+config:
+  google:
+    client_id: '1091187390014-u2l759rm5nu28ujik0glt6l214g5abm9.apps.googleusercontent.com'
+    client_secret: 'Bkxy_IQOVsXBAbTIuJkVzpLM'
+    domain: sendbirdtest.com
+
+webserver:
+  webserverConfig: |
+    # -*- coding: utf-8 -*-
```
Comment (JB-SendBird) [inline chart/values_nightly.yaml:18]: This configuration python file is necessary if we use RBAC of Airflow. We could extract this as a separate file and include it in our docker image further, under AIRFLOW_HOME path.

File: chart/values_nightly.yaml [hunk 4]
```diff
@@ -0,0 +1,126 @@
+defaultAirflowRepository: 242864343139.dkr.ecr.ap-northeast-1.amazonaws.com/sendbird/airflow
+defaultAirflowTag: v0.0.1-dev
```
Comment (JB-SendBird) [inline chart/values_nightly.yaml:2]: Built a new docker image file based on `apache/airflow:1.10.14`, including `Flask-OAuthlib` which is necessary for using Google OAuth login.

Comment (JB-SendBird) [reply chart/values_nightly.yaml:2]: The image was just for dev purpose. We will create a repo for DAGs and build images there and use it.

**Pull request merged**